postgres-appliance/configure_spilo.py

```diff
@@ -219,14 +217,21 @@ def get_instance_metadata(provider):
                 'id': socket.gethostname(),
                 'zone': 'local'}
 
+    if USE_K8S:
```

Comment:
For naming consistency, you have `KUBE` above, `K8S` here, maybe we should be somehow consistent. Do you think that `KUBERNETES` would be too long?